src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -126,70 +126,118 @@ void AppHost::LastTabClosed(const winrt::Windows::Foundation::IInspectable& /*se
 //   create. We'll use this rect to determine which monitor the window is about
 //   to appear on.
 // Return Value:
-// - <none>
-void AppHost::_HandleCreateWindow(const HWND hwnd, const RECT proposedRect)
+// - A LaunchMode enum that indicates the launch mode
+winrt::TerminalApp::LaunchMode AppHost::_HandleCreateWindow(const HWND hwnd, RECT proposedRect)

```

Comment:
I don't know how to feel about returning the launch mode here. Was this a @DHowett-MSFT or @zadjii-msft request? It seems a little odd to me.

Reply:
Changed. Now pass LaunchMode as reference to _HandleCreateWindow.